recipe/meta.yaml

```diff
@@ -23,7 +23,7 @@ requirements:
     - gdal 2.1.*
     - ghostscript
     - libnetcdf 4.4.*
```

Comment:
>  Is there a way for me to re-trigger the CIs without amending the commit?

Closing and re-opening usually does the trick. If not just ping me and I will restart the CIs.